packages/vue/src/examples/pages/checkout/_internal/Shipping.vue

```diff
@@ -73,7 +73,7 @@
         v-model="country"
         :value="country"
         label="Country"
-        class="form__element form__element--half form__element--half-even form__select sf-select--underlined"
+        class="form__element form__element--half form__element--half-even select sf-select--underlined"
```

Comment:
@AdamPawlinski do we need this change? why did you add height on select? :)

Reply:
without that it is smaller than inputs and is moved down.